helpers/README.md

```diff
@@ -126,6 +126,9 @@ getComposedActiveElement()
 // gets the first focusable descendant given a node, including those within the shadow DOM
 getFirstFocusableDescendant(node, includeHidden, predicate, includeTabbablesOnly)
 
+// gets the focusable elements within the specified element
```

Comment:
Not suggesting any specific changes (or a timeline), but I do want to note that there may be some cleanup to do here to align some of the names and descriptions better e.g. `getFocusables` vs `getComposedChildren`, "descendants" vs "children", "element" vs "node".

Like, should this be `getFocusableChildren`, does it get "the focusable descendants of a given node" etc.?

Reply:
Good point - I think this should be named `getFocusableDescendants` to be consistent with other usage in these helpers. For instances in this module where `Children` is used, it refers to direct element descendants and was originally chosen to be consistent with the browser APIs. It carries a different meaning than `Descendants` and I think it is accurate.

In several places (not just this module), `node` and `elem` have been used interchangeably where we should have been more intentional. In this method, an element is expected and I will clean this up.

Reply:
Except that the `deep` option (default `false`) is what actually determines if it gets just children or all descendants?

Reply:
That's true. The method can return either direct or indirect "descendants". I feel like `Descendants` can be used generically in that sense, but if we feel that `Descendants` should always refers to all direct and indirect, then maybe what I originally had was best.

Reply:
Btw, I decided to keep `node` - it's actually accurate since it can technically be an element, a document, or a fragment.